### PR TITLE
Adding PropTypes to Components

### PR DESCRIPTION
**Pull request by hereisnaman**
Adding proptypes for components.

Have completed:

HistoryItem https://github.com/iodide-project/iodide/issues/296
TwoRowCell https://github.com/iodide-project/iodide/issues/289
CellContainer https://github.com/iodide-project/iodide/issues/290
CellEditor https://github.com/iodide-project/iodide/issues/291
CellRow https://github.com/iodide-project/iodide/issues/292
CellTypeExternalResource https://github.com/iodide-project/iodide/issues/293
Working on adding more.

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@hereisnaman I am also adding some PropTypes. I have added output.jsx and two-row-cell.jsx. Since you didn't commented on issue so i thought no body is working on that. I think we can divide the components we are adding to avoid duplicate work. :)

**Comment by hereisnaman** [comment]
@pradeepgangwar from issue 298 to 303 are remaining. Can you work on them if you want to, all others I have done.

**Comment by hereisnaman** [comment]
@pradeepgangwar Also just send a PR to my branch so that commits of both of us can get merged under one PR.

**Comment by pradeepgangwar** [comment]
But i have opened another PR. I didn't notice this one before. So let me push to that only.

**Comment by hereisnaman** [comment]
Okay cool.

File: .gitignore [hunk 1]
```diff
@@ -62,4 +62,7 @@ typings/
 # dotenv environment variables file
 .env
 ext_resources/
-build/
```
Comment (pradeepgangwar) [inline .gitignore]: @hereisnaman I think you deleted this by mistake. Actually we need this in .gitignore. :)

Comment (hereisnaman) [reply .gitignore]: I think you need to recheck that it is the first rule in the file. It was repeated at the bottom. So i removed one.

**Comment by bcolloran** [comment]
Thanks so much Naman and Pradeep! Looks great!

**Pull request merged**